modules/@apostrophecms/area/index.js

```diff
@@ -636,6 +655,17 @@ module.exports = {
           }
         });
       },
+      addWidgetOperation(operation) {
+        if (!operation.label || !operation.modal) {
+          throw self.apos.error('invalid', 'addWidgetOperation requires label and modal properties.');
+        }
+
+        self.widgetOperations = self.widgetOperations.filter(
+          ({ modal }) => modal !== operation.modal
```

Comment:
We're implementing overrides based on a different modal name? We might want to add support for `props`, which could open the door to different uses of the same modal. So if we really need this at all right now, then overrides by operation `name` would make more sense I think.